osu.Game.Rulesets.Osu/Edit/OsuSelectionHandler.cs

```diff
@@ -13,7 +13,7 @@ public class OsuSelectionHandler : SelectionHandler
     {
         public override void HandleDrag(SelectionBlueprint blueprint, DragEvent dragEvent)
         {
-            foreach (var h in SelectedHitObjects.OfType<OsuHitObject>())
+            foreach (OsuHitObject h in SelectedHitObjects.OfType<OsuHitObject>())
```

Comment:
is there a problem with using `var` here? it can be clearly seen that the type is `OsuHitObject` because of the filtering `Enumerable.OfType<FooType>()`

Reply:
To make it clear it is a hit object, because "var h" is not a clear enough

Reply:
it's made clear by the source in this case.

Reply:
please don't resolve conversations